### PR TITLE
[AWS] Update all WATT instances

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] N/A
[Problem] We are maintaing two WATT instances for Seoul and Mumbai regions
          but watt-aws-update.sh updates only one.
[Solution] Change watt-aws-updates.sh script to update both instance.
           In addtion, change watt-aws-updates.sh to watt-aws-instances-update.sh.
           Update README.md with proposed changes.

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

**Comment by grzegorz-czajkowski** [comment]
I will verify the patch during the nearest code.tizen.org update.

**Comment by grzegorz-czajkowski** [comment]
I verified the script. It successfully updates both instances on AWS.

File: watt-aws-instances-update.sh [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    docker tag $local_watt_image $watt_docker_aws_repository_uri
+    if [ $? -ne 0 ]; then
+        echo "Could not tag images"
+        exit 1
+    fi
+
+    # Login to aws.
+    aws ecr get-login --no-include-email --region $region | bash -
+    if [ $? -ne 0 ]; then
+        echo "Could not login"
```
Comment (gwolny) [inline watt-aws-instances-update.sh]: what about giving info about the region

File: watt-aws-instances-update.sh [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        exit 1
+    fi
+
+    # Push the image to repository Uri.
+    docker push $watt_docker_aws_repository_uri
+    if [ $? -ne 0 ]; then
+        echo "Could not push images"
```
Comment (gwolny) [inline watt-aws-instances-update.sh]: ditto

File: watt-aws-instances-update.sh (hunk 1 above)
Comment (grzegorz-czajkowski) [reply watt-aws-instances-update.sh]: Done.

File: watt-aws-instances-update.sh (hunk 2 above)
Comment (grzegorz-czajkowski) [reply watt-aws-instances-update.sh]: Done.

**Pull request merged**